packages/flutter_hooks/lib/src/framework.dart

```diff
@@ -417,6 +447,21 @@ mixin HookElement on ComponentElement {
       _buildCache = super.build();
     } finally {
       _isOptionalRebuild = null;
+
+      for (final hook in _needDidBuildHooks) {
```

Comment:
Will need to be wrapped into an `if (_needDidBuildHooks != null)` first

I'd also consider using `for (var i = 0; i < hook.length; i++)`, which should be slightly faster for a List.

Reply:
`if` is added but as I kept the the `Set` I've not change the `for`, let me know